roles/nighthawk/templates/workload.yml.j2

```diff
@@ -0,0 +1,135 @@
+---
+kind: Job
+apiVersion: batch/v1
+metadata:
+{% if workload_args.serviceip is sameas true %}
+{% if workload_args.service_type | default("ClusterIP") == "NodePort" %}
+  name: 'nighthawk-client-{{item.status.hostIP}}-{{ item.metadata.labels.app }}-{{ trunc_uuid }}'
+{% endif %}
+{% else %}
+  name: 'nighthawk-client-{{item.status.podIP}}-{{ trunc_uuid }}'
+{% endif %}
```

Comment:
```suggestion
  name: 'nighthawk-client-{{item.status.podIP}}-{{ trunc_uuid }}'
```

Reply:
Keeping the name in the "if" clause as it more verbose and understandable while debugging.